src/FactoryPattern/FactorySelection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FactoryPattern
+{
+    class FactorySelection
+    {
+        public static CarModelFactory CreateCarFactory(string FactoryType)
+        {
+            if (FactoryType.ToLower() == "electric")
+            {
+                System.Console.WriteLine("entrel meqenaneric meke |Tesla,Nissan,Jaguar|");
```

Comment:
Dont hardcode the cars list `Tesla,Nissan,Jaguar`. Let factory decide which cars it will produce. I.e. by `carFactory.getCars()`